x/params/gas/ante.go

```diff
@@ -0,0 +1,40 @@
+package gas
```

Comment:
If feasible, can we move making gas use the param store into a separate PR? This PR is already quite big, and this could likeley be reviewed independently.